### ISSUE TITLE
Cache duration does not work; cache is always set to 12 hours

### ISSUE THREAD

**Issue by contactashish13** [Codeinwp/feedzy-rss-feeds#422]
ref: https://wordpress.org/support/topic/feed-refresh-rate-not-being-applied/

No matter what the value of `refresh` is, the cache duration in the database is always set to 12 hours. 

Investigation has revealed that SimplePie's `set_cache_duration` does not work for DB caching.

**Comment by contactashish13** [comment]
PR https://github.com/Codeinwp/feedzy-rss-feeds/pull/423 fixes this. 

The only way to test this is to check the database entries as mentioned in the thread: https://wordpress.org/support/topic/feed-refresh-rate-not-being-applied/

**Comment by rodica-andronache** [comment]
I'm not sure if I tested properly, but here it is.
I've used this plugin to see the transients https://wordpress.org/plugins/transients-manager/

For shortcodes, it looks to work ok. I used `[feedzy-rss feeds="https://s3.amazonaws.com/downloads.themeisle.com/sample-feed-import.xml" refresh="1_hours"]` and the transient was set to expire in 60 minutes.

But for the feedzy block, I entered a feed, selected 2 Hours in the option for Feed Caching time, and went ahead to see the post. At that moment, the transient is set to 12 hours.
Only if I go and manually delete the transients, and revisit the posting page, the transient gets the correct value.
https://www.loom.com/share/6fc05952b47f46f6825f97943921e868
